create_output.py

```diff
@@ -63,19 +63,19 @@ def create(file_name, receive_sensitivity, definition):
         print 'Failed converting ppm to png'
     try:
         print 'Converting white to transparency'
-        #os.system(w2t)
+        os.system(w2t)
```

Comment:
You've had this commented out @gotofftherails, can you tell me why? Do they not work?